schemes/hack_shallow/cmfmca_namelist.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+
+<?xml-stylesheet type="text/xsl"?>
+
+<entry_id_pg version="2.0">
+  <entry id="hkconv_cmftau">
+    <type>real</type>
+    <category>conv</category>
+    <group>hkconv_nl</group>
+    <standard_name>characteristic_adjustment_time_for_shallow_convection_tbd</standard_name>
+    <units>s</units>
+    <desc>
+      Characteristic adjustment time scale for Hack shallow scheme. Default: 1800.0
+    </desc>
+    <values>
+      <value>1800.0</value>
+    </values>
+  </entry>
+  <entry id="hkconv_c0">
+    <type>real</type>
+    <category>conv</category>
+    <group>hkconv_nl</group>
+    <standard_name>rain_water_autoconversion_coefficient_for_shallow_convection_tbd</standard_name>
+    <units>m-1</units>
+    <desc>
+      Rain water autoconversion coefficient for Hack shallow scheme.
+    </desc>
+    <values>
+      <!-- FIXME: This is a resolution-dependent value -->
+      <value>1.0e-4</value>
```

Comment:
is this a "FIXME before merging this PR" or a "FIXME eventually when we have some capability? If it's the latter, I'd suggest a comment describing what we need to implement

Reply:
Good question... it was more of a reminder that eventually the resolution-dependent options need to be ported, but from the [CAM-SIMA docs on namelist XML files](https://escomp.github.io/CAM-SIMA-docs/conversion/create-namelist-xml/) it looks like this work is implied: so I got rid of the comment.

> When populating the value fields, you don't need to port all of them at this time. You need to put in the value that is used by default by CAM, but the rest will be ported once all schemes have been ported.